1_fetch/src/get_nwis_data.R

```diff
@@ -109,47 +109,36 @@ get_peak_flow_log <- function(files_in, file_out) {
 prescreen_daily_data <- function(filename, prov_rm = TRUE){
   #loads data from file and removes provisional and estimated data if prov_rm = TRUE
   message('loading ', filename)
-  ##Handle sites with strange column names
-  if (length(grep(pattern = '01011500', filename)) +
-      length(grep(pattern = '02196000', filename)) + 
-      length(grep(pattern = '03213000', filename)) + 
-      length(grep(pattern = '12010000', filename)) > 0){
+  d0 <- read_csv(filename,
+                 name_repair = "minimal", 
+                 col_types = cols(agency_cd = col_character(), 
+                                  site_no = col_character(), Date = col_date(format = "%Y-%m-%d"), 
+                                  discharge = col_number(), discharge_cd = col_character(), 
+                                  dateTime = col_character(), tz_cd = col_character())) %>%
```

Comment:
It only happened for a single site - 04153500. It is completely blank. All other sites have either 5 or 7 columns. I had to add the dateTime and tz_cd columns to make sure the class of those columns would play nice with the other matrices.

Reply:
04153500 was not available before, and I see that it doesn't have a discharge column. Weird that it would be downloaded at all if it has no flow data. It might be omitted from the log file because the column names don't match the other datasets

Reply:
@slevin75 We did have an issue with 04153500 before, see above. Before these changes were pushed, "04153500_dv.csv" only contained column headers (and not the standard) and no data. Does your "01_fetch/out/04153500_dv.csv" exist, and if so, what does it contain?

Reply:
> ch/out/04153500_dv.csv" exist, and if so, what does it contain?

It is just an empty csv file, there's nothing in it.  I think it is because data for this site doesn't start until after our end date, so it downloads nothing.  We probably just need to add an additional filter to the has_data_check  function to make sure that the data begin prior to our end date.

Reply:
I believe we already added those filters. Maybe you need to re-run the pipelines for has_data and prescreen_daily, etc.?

Reply:
Because when I re-ran all p1_ targets after this merge, I didn't have an error for 04153500 (because it never generated the .csv).

Reply:
huh. ok I see it now in my forked repo.  I think I did not do git correctly.  Why is git so confusing to me? I see the changes in my forked repo but not my local.

Reply:
Just thought of something, if you re-ran the early part of the pipeline without deleting all the .csv outputs, it would probably overwrite the sites that "passed" the latest 'has_data()' and 'prescreen_()' functions, but the old sites would remain in the folder. I deleted all of the files and then re-ran the pipeline, which is what ensured that 04153500 was deleted from my list of .csv files. 

Or... I'm not correctly understanding how targets works, which is likely...

Reply:
Git is confusing... 

A few weeks ago I updated my R version. When I did that, I completely deleted my local repo (on my computer's file explorer) then re-cloned the main branch from our USGS repo homepage. A full, clean reset. That might help, but it will take a while to re-run the pipeline!

Reply:
@cstillwellusgs no, I think I just messed up in git.  I don't see the extra filter statement in my local version.  I did a fetch upstream and updated my forked repo, but I guess I did not correctly synch it to my local machine.

Reply:
I have to open this every time I do something in git for this project. Maybe something in here will help. 

https://teams.microsoft.com/l/file/52604EAA-E543-4D2E-B330-89B4BCC7D012?tenantId=0693b5ba-4b18-4d7b-9341-f32f400a5494&fileType=docx&objectUrl=https%3A%2F%2Fdoimspp.sharepoint.com%2Fsites%2FFHWAhydrologicforcingsandchannels%2FShared%20Documents%2FGeneral%2FPhase%201%20plans%2FCodeWorkflow%2BDataManagementPlan.docx&baseUrl=https%3A%2F%2Fdoimspp.sharepoint.com%2Fsites%2FFHWAhydrologicforcingsandchannels&serviceName=teams&threadId=19:3938896a850343718c646ad575fd71e8@thread.skype&groupId=807bdbb6-f308-47a5-ac2a-ed76608656f3